lib/icon.js

```diff
@@ -12,6 +13,7 @@ const IconBase = styled.img`
   display: inline-block;
   height: 1em;
   width: auto;
+  vertical-align: top;
```

Comment:
this is required for the buttons with icons to line up correctly with those that are without icons.